INSTALL.md

```diff
@@ -0,0 +1,136 @@
+# Installing client-go
+
+## For the casual user
```

Comment:
Chao is removing such dependencies (k8s.io/apimachinery & glog for starters) from vendor/